portal/src/texts/profile/a11y.md

```diff
@@ -1,8 +1,10 @@
 ## Tilgjengelighet
 
-Universell utforming handler om at vi skal lage løsningene våre med tanke på at brukere kan ha ulik funksjonsevne, midlertidig eller varig. Når vi tenker på å utforme tilgjengelige løsninger, når vi alle målgruppene med én og samme løsning.
+Universell utforming handler om at vi lager løsninger med tanke på at brukere kan ha ulik funksjonsevne. Den kan være  midlertidig eller varig. Eksempler kan være synshemming, nedsatt finmotorikk eller vansker med å skille farger. Når vi planlegger hvordan vi skal utforme løsninger som er tilgjengelig for alle, får vi én løsning som når alle målgrupper.
```

Comment:
How about `Universell utforming handler om at vi lager løsninger for alle`, that at least what a11y is to me. Not really the special handling for special needs, but rather common sense that everyone should be able to access our stuff.

Reply:
hva med "Universell utforming handler om at vi lager gode løsninger for alle". Men mener du at vi da skal ta ut eksemplene på funksjonsnedsettelser? Når jeg jobber med ulike folk her på huset, er det ikke bevisste på at tekst i lenker for eksempel skal kunne leses opp av en skjermleser, så da kan det være viktig med eksempler i Jøkul. Hva tror du?

Reply:
Good accessibility makes things easier to use for _everyone_, not just those with disabilities. We should definitely make people aware that people with disabilities use our services, but the focus should be that we make the solutions _better_ (just plain better) by having good accessibility 😊

Reply:
Maybe something like
`Universell utforming handler om å lage løsninger som er lette å bruke for alle. Våre brukere kan ha ulik funksjonsevne. Den kan være midlertidig eller varig nedsatt. Eksempler er synshemming, nedsatt finmotorikk, og nedsatte kognitive evner. Når vi utformer og utvikler løsninger som er tilgjengelige for disse brukerne, får vi løsninger som er bedre for alle målgrupper.`

Reply:
Kjøpt @piofinn :-)